docs(android-core): chat and stage v2 changes (MOB-1905, MOB-1908) (#406)
dyte-io/docs: docs/android-core/chat/sending-a-chat-message.mdx

sends it to the participants in the meeting.

```kotlin
meeting.chat.sendImageMessage(imageUri) { err ->
  // Handle error if any
}
```

## Send a file

Sending a file is quite similar to sending an image. The only difference is that when you send an image, a preview will be shown in the meeting chat, which is not the case for sending files. That being said, an image can be sent as a file too using `meeting.chat.sendFileMessage()`.

```kotlin
meeting.chat.sendFileMessage(fileUri) { err ->
  // Handle error if any
}
```

> ## Error types

Reviewer: ```suggestion
### Chat Errors
```

> The `sendTextMessage` method does not have a callback function, so it return an error object of type `ChatTextError`, `null` if there's no error. It can be the following:

Reviewer: ```suggestion
The `sendTextMessage` method returns a `ChatTextError` if the operation fails, `null` if successful. The error can be the following:
```
